src/test/java/org/folio/context/PostgresTestContainersBeanFactoryPostProcessor.java

```diff
@@ -1,4 +1,4 @@
-package org.folio.consortia.context;
+package org.folio.context;
```

Comment:
Lets remain in this package org.folio.consortia , because it is common package for all functionality in this module

Reply:
done